website/docs/concepts/modifiers/family.mdx

```diff
@@ -101,7 +101,18 @@ This includes:
 - Objects generated with [Freezed] or [built_value](https://pub.dev/packages/built_value)
 - Objects using [equatable](https://pub.dev/packages/equatable)
 
-Here's an example using [Freezed]:
+Here's an example using [Freezed] and [equatable]:
```

Comment:
the [equatable] link is broken

Reply:
"Here's an example using [freezed]" should be inside the Freezed tab, not above all tabs

Reply:
Added the link equatable